### DISCUSSION TITLE
Software or algorithms for fitting geometry to mesh vertices

### DISCUSSION THREAD

**Discussion by Bald888Eagle** [markqvist/Reticulum#247] [Help & Questions]
Hi Mark,
I'm a computer-graphics enthusiast who typically works out their own algorithms and macros and works things out from first principles. I typically deal with mathematical primitives such as planes, cylinders, cones and spheres, but also extrapolate this work to Bezier curves and surfaces, and have worked out how to maintain things like C2 and G2 curvature, and calculate the mean and Gaussian curvature of those mathematically well-behaved surfaces, 
However, I'm interested in processing an arbitrary mesh with a large number of vertices - say, 77,000 or so - and wanted to know if you had any recommendations or simple diagrams or examples on how to go about fitting shapes and tangent lines to features of the mesh.
(I also frequently find myself using ratios of irrational numbers due to expressing angles in radians vs degrees, and was wondering if there was a convenient way to identify and highlight such "even" fractional values of mathematical constants like tau.)

I just lost my last 2 Linux systems and am stuck using a Win 7 box for the time being.  :(

Thank you very much for your time,

Bill Walker
creativechemist netscape net
connect the dots and fill in the blanks  ;)

**Comment by markqvist** [comment]
Hi Bill!

> and have worked out how to maintain things like C2 and G2 curvature, and calculate the mean and Gaussian curvature of those mathematically well-behaved surfaces

We need to talk!

I'll send you a proper reply by mail tomorrow.

I have some hardware well suited for such tasks, set up for remote operation, by the way :)

**Comment by Bald888Eagle** [comment]
Thanks Mark, I sent you an email.
